shanoir-ng-front/src/app/import/dicom-upload/dicom-upload.component.ts

```diff
@@ -51,8 +51,13 @@ export class DicomUploadComponent {
     }
     
     private uploadArchive(fileEvent: any): void {
-        this.setArchiveStatus('uploading');
-        this.uploadToServer(fileEvent.target.files);
+        if (fileEvent.target.files.length > 0) {

```

Comment:
This code could be moved into a single file, but I think it should be done in another issue where we refactor import in general.